WcaOnRails/spec/models/registration_spec.rb

```diff
@@ -54,13 +54,13 @@
   end
 
   it "requires at least one event" do
-    registration.eventIds = ""
+    registration.registration_events.destroy_all
```

Comment:
This sounds good! I was worried that the immediate database write would be a problem, but I think we're okay here because our controllers call `update_attributes` which runs entirely in a transaction =)